acceptance/setup/common/010_Setup_Broker.rb

```diff
@@ -10,5 +10,11 @@
 step 'Run lein once so it sets itself up'
 on master, 'chmod a+x /usr/bin/lein && export LEIN_ROOT=ok && /usr/bin/lein'
 
-step 'Run pcp-broker in trapperkeeper in background (return immediately)'
-on master, 'cd ~/pcp-broker; export LEIN_ROOT=ok; lein tk </dev/null >/dev/null 2>&1 &'
+step 'Run lein deps to download dependencies'
+# 'lein tk' will download dependencies automatically, but downloading them will take
+# some time and will eat into the polling period we allow for the broker to start
+on master, 'cd ~/pcp-broker; export LEIN_ROOT=ok; lein deps'
+
+step 'Run pcp-broker in trapperkeeper in background and wait for port 8142'
+on master, 'cd ~/pcp-broker; export LEIN_ROOT=ok; lein tk </dev/null >/var/log/pcp-broker.log 2>&1 &'
+retry_on(master, "ss -nap | grep 'LISTEN.*8142'", {:max_retries => 60})
```

Comment:
Possibly better to use `port_open_within?(master, 8142, 120)` https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/helpers/web_helpers.rb#L9-L13